test/ExecutionFramework.h

```diff
@@ -53,7 +53,7 @@ class ExecutionFramework
 
 public:
 	ExecutionFramework();
-	explicit ExecutionFramework(std::string const& _ipcPath);
+	explicit ExecutionFramework(std::string const& _ipcPath, langutil::EVMVersion const _evmVersion);
```

Comment:
Value arguments should not contain the const modifier in function signatures, as they're copied by value, and the callee has thus its own copy, and the caller should not care about whether or not the callee is modifying the copied value on the callee-site or not.

That at least was the argument in one of the talks I have been watching by Herb Sutter and I hoped to have it also read in the CppCoreGuidelines, but can't find it right now.

I still think the `const` should not be there, do you? :)

Reply:
I don't think ``const`` should be on any of them, no - I put it in here just because the surrounding code has them all over the place - not sure where they came from :-). I'd be happy for someone to remove all of them, though :-).